.changeset/real-yaks-press.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`SideNav` - Deprecated the `SideNav` component. Use the [`AppSideNav` component](/components/app-side-nav) as a replacement.
```

Comment:
[question] should we mention something about the fact that we have re-added back the enterprise navigation components? This would also be visible in the release notes, and could be helpful to have a reference of when they were added back.